src/App.css

```diff
@@ -1,24 +1,83 @@
-.App {
+.Title {
+  display:flex;
+  flex-direction: column;
+  align-items: stretch;
   text-align: center;
+  background-color: #00ffff;
+  padding-top:80px;
```

Comment:
Yeah, we can use flexbox to do this kind of vertical centering, let's do that after merging